DSharpPlus/Clients/DiscordClient.cs

```diff
@@ -619,19 +619,27 @@ private void UpdateMessage(DiscordMessage message, TransportUser author, Discord
 
             var channel = this.InternalGetCachedChannel(message.ChannelId);
 
-            if (channel == null)
+            if (channel != null) return;
+            
+            if (!message.GuildId.HasValue)
             {
+                channel = new DiscordDmChannel
+                {
+                    Id = message.ChannelId,
+                    Discord = this,
+                    Type = ChannelType.Private
+                };
+                
+            }
+            else {
```

Comment:
Brace styling should be under the else.

Reply:
I wrote this in VS (which I recently re-installed), so I guess my code style got wiped along with it

Reply:
Fixed braces and unnecessary whitespace 👍🏽